pkg/controller/nmstate/nmstate_controller.go

```diff
@@ -2,6 +2,7 @@ package nmstate
 
 import (
 	"context"
+	helper "github.com/nmstate/kubernetes-nmstate/pkg/helper/render"
```

Comment:
please move to other import group

Reply:
Yep remove it from helper `pkg/render` should be

Reply:
Excude me, I'm a bit lost here, do you mean moving the import into separated group of imports from the same repo?
like:
```
import (
	"context"
        . . . . 

	helper "github.com/nmstate/kubernetes-nmstate/pkg/helper/render"
	nmstatev1beta1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1beta1"
	"github.com/nmstate/kubernetes-nmstate/pkg/names"

	"github.com/openshift/cluster-network-operator/pkg/apply"
       . . . .
```